src/Image/style.css

```diff
@@ -0,0 +1,33 @@
+@import 'antd/lib/image/style/index.css';
+@import 'antd/lib/style/default.css';
+
```

Comment:
Import order is important. Always import styles in the same order as the library. If you see `\antd\lib\image\style\css.js`, `default` style comes before `image` style

Reply:
ahh okay noted. will do the necessary changes